charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -66,7 +65,13 @@ spec:
         release: {{ $root.Release.Name }}
         component: terminating-gateway
         terminating-gateway-name: {{ template "consul.fullname" $root }}-{{ .name }}
+        consul.hashicorp.com/connect-inject-managed-by: consul-k8s-endpoints-controller
       annotations:
+        "consul.hashicorp.com/connect-inject": "false"
```

Comment:
btw don't know if you saw it, but this is a duplicate annotation. It is already there after all the vault stuff

Reply:
Oh geez, good catch. I didn't see that. Thank you!